pkg/windows/windows.go

```diff
@@ -507,6 +539,22 @@ func (vm *windows) ConfigureAzureCloudNodeManager(nodeName string) error {
 	return nil
 }
 
+// UpdateKubeletClientCA updates the kubelet client CA certificate file in the Windows node. No service restart or
+// reboot required, kubelet detects the changes in the file system and use the new CA certificate. The file is replaced
+// if and only if it does not exist or there is a checksum mismatch.
+func (vm *windows) UpdateKubeletClientCA(contents []byte) error {
```

Comment:
To me, this needs to be a NodeConfig method as it is not performing a generic Windows VM operation, and instead performing an operation directly related to ensuring the health of a Windows Node.

Reply:
Roger that!